Classes/YLRefreshHeaderView/YLRefreshHeaderView.m

```diff
@@ -83,4 +98,4 @@ - (void)containingScrollViewDidEndDragging:(UIScrollView *)scrollView {
   }
```

Comment:
In here and in the method below, scrollView.contentOffset.y is used to determine whether we pulled far enough to refresh. However, if the scrollview has a contentInset > 0, scrollView.contentOffset will directly be -scrollView.contentInset.top. You thus need a helper method to give you the relativeContentOffset of the scrollview that will abstract the contentInset part of things, something like "self.scrollView.contentInset.top + self.scrollView.contentOffset.y";
You can then replace every place using the scrollView.contentOffset.y with this and it should work.